src/tmle_inputs/tmle_inputs.jl

```diff
@@ -10,13 +16,13 @@ The SAMPLE_ID column should be read as a String.
 """
 read_data(filepath) = CSV.read(filepath, DataFrame, types=Dict(:SAMPLE_ID => String))
 
-function write_tmle_inputs(outprefix, final_dataset, parameters; batch_size=nothing)
+function write_tmle_inputs(outprefix, final_dataset, parameters; tf_name=nothing, batch_size=nothing)
```

Comment:
why did you need to change that function? I don't think it is called in the from_actors mode

Reply:
I updated this because this function write_tmle_inputs() is also called from the function tmle_inputs_from_param_files(), and I needed to set a default tf_name given the way I organized it. I think if we make TF optional as mentioned above this can be reverted.

Reply:
reverted back to original: 219f947a0878e0aacd61a8f7e72730c31d356660